tests/ui/enum/nested-enum.stderr

```diff
@@ -2,25 +2,37 @@ error: `enum` definition cannot be nested inside `enum`
   --> $DIR/nested-enum.rs:2:5
    |
 LL |     enum Bar { Baz },
-   |     ^^^^------------
-   |     |
-   |     help: consider creating a new `enum` definition instead of nesting
+   |     ^^^^
+   |
+help: consider creating a new `enum` definition instead of nesting
+   |
+LL -     enum Bar { Baz },
+LL +
+   |
 
 error: `struct` definition cannot be nested inside `enum`
   --> $DIR/nested-enum.rs:3:5
    |
 LL |     struct Quux { field: u8 },
-   |     ^^^^^^-------------------
-   |     |
-   |     help: consider creating a new `struct` definition instead of nesting
+   |     ^^^^^^
+   |
+help: consider creating a new `struct` definition instead of nesting
+   |
+LL -     struct Quux { field: u8 },
+LL +
```

Comment:
uhm?

Reply:
Pre-existing. It happens when we suggest removing something but leave whitespace behind. I know I fixed this in a separate PR that might not have been merged yet, but can't find the commit 😵‍💫

Reply:
Oh, I just found what the problem: `//~ ERROR` is removed after the stderr file is created, the suggestion makes more sense when running against the file manually.